docs/canvas/canvas-function-reference.asciidoc

```diff
@@ -2134,19 +1996,21 @@ Renders the _context_ as a specific element and sets element level options, such
 
 |`as`
 |`string`
-|The element type to render. You might want to use a specialized function instead, such as <<plot_fn>> or <<shape_fn>>.
-
-|`css`
-|`string`
-|Any block of custom CSS to be scoped to the element 
-
-Default: `".canvasRenderEl{\n\n}"`
+|The element type to render. You probably want a specialized function instead, such as <<plot_fn>> or <<shape_fn>>.
```

Comment:
We should probably go into detail in the expression language docs instead of in this help text. Basically if I have a `datatable` and I want to add a `render as="plot"` for instance at the end, it won't work without using the specialized `plot` function.